extensions/myanimelist-search/src/components/listDetail.tsx

```diff
@@ -0,0 +1,27 @@
+import { List } from "@raycast/api";
+import { ExtendedAnime } from "../api/api";
+
+export default function AnimeDetails({ anime }: { anime: ExtendedAnime }) {
+  const startYear = anime.start_date ? new Date(anime.start_date).getFullYear().toString() : "-";
+
+  return (
+    <List.Item.Detail
+      markdown={`## ${anime.title}\n<img src="${anime.main_picture.large}" height="200"/>\n\n${anime.synopsis || ""}`}
+      metadata={
+        <List.Item.Detail.Metadata>
+          <List.Item.Detail.Metadata.Label title="English Title" text={anime.title || "-"} />
+          <List.Item.Detail.Metadata.Label title="Type" text={anime.media_type || "-"} />
+          <List.Item.Detail.Metadata.Label title="Episodes" text={anime.num_episodes.toString() || "-"} />
```

Comment:
change to `text={anime.num_episodes > 0 ? anime.num_episodes.toString() : "-"}` so ongoing animes, that don't have a number of episodes, also return "-"